templates/base/src/utils/Logger/logger.js

```diff
@@ -0,0 +1,125 @@
+
+import fetch from 'isomorphic-unfetch';
+
+/**
```

Comment:
Instead of commenting it here, lets add an MD file with Examples.

Reply:
done